src/lib/constants/Messages.ts

```diff
@@ -36,4 +36,19 @@ export const Messages = {
       'vtex workspace create <workspace> --production'
     )} to create a production workspace`,
   PROMOTE_SPINNER_START: 'Preparing the workspace to be promoted',
+  PROMOTE_CONFLICT_ERROR: (workspace: string, buckets: string[]) =>
+    `${
+      `Couldn't promote workspace ${ColorifyConstants.ID(workspace)}.\n` +
```

Comment:
What I need to ask you to make different, as a designer: 

- Remove the time information that comes along the error message;
- The "action" the message suggests is the first information the user wants to read so, we bring it to the first sentence and explain the context and why to do what we are suggesting after the action. It gives more context to the user decision flow; 
- The instructions are a little bit long but I'm happy we are prioritizing information insurance over simplicity for now (we may discover it is not the best furthermore, but for now sounds safe hahaha), to reduce the sensation of "too much to read", we should break the content with line dividers. 

I assume that is not the ideal flow to me yet. In the ideal scenario, we identify the conflict, ask the user if she wants us to solve it OR if she wants to check it out by herself. If she says "yes" for the Toolbelt "auto-solving" we run a script that considers the app context and automatically sends a request to the workspace and it's all ready to be promoted. 

As I understand maybe `promote` issues are not such a big deal right now, I'm just sharing how it could work to be more aligned with a more proactive Toolbelt :) 

[Check it out on Figma](https://www.figma.com/file/GpNy0htTpJ7uvw7WKqGdLH/Toolbelt-DX-review?node-id=613%3A282)
![image](https://user-images.githubusercontent.com/19560449/93605605-589bab00-f99d-11ea-85b9-173116b0f2eb.png)

Reply:
Thank you for the comments!
They were spot on.

I'd love to have more time to create the best experience possible here, but the backlog is huge right now.

I've almost copied and pasted the sentences you've used.
I considered removing the `This request will trigger a rebase process on your code`, but after we've spoken through Slack I've chosen a different sentence to communicate a similar idea. 

Feel free to make any suggestions and request changes :)